.github/workflows/test_released_action.yml

```diff
@@ -19,7 +19,30 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: CCS Build action step
-        uses: apollo-fire/ccs-build@v0.0.5
+      - name: CCS Build action step - Release
+        uses: apollo-fire/ccs-build@v0.0.6
         with:
           project-path: 'test/working_ccs_project'
+          project-name: 'msp430_test_project'
+          build-configuration: Release
+
+      - name: CCS Build action step - Debug
+        uses: apollo-fire/ccs-build@v0.0.6
+        with:
+          project-path: 'test/working_ccs_project'
+          project-name: 'msp430_test_project'
+          build-configuration: Debug
+
+      - name: Upload release firmware artifact
+        uses: actions/upload-artifact@v4
+        with:
```

Comment:
You can maximise the compression level for the ti-txt files
https://github.com/actions/upload-artifact?tab=readme-ov-file#altering-compressions-level-speed-v-size